lib/editor/components/ScheduleExceptionForm.js

```diff
@@ -150,7 +150,7 @@ export default class ScheduleExceptionForm extends Component<Props> {
               <small>Exception name*</small>
             </ControlLabel>
             <FormControl
-              value={name}
+              value={!name ? '' : name}
```

Comment:
You should be able to reduce this a bit and I would add a comment to explain. 
```suggestion
              value={name || ''} // If name not given, provide empty string to override FormControl state.
```